streams/shared/src/main/scala/zio/stream/ZStreamChunk.scala

```diff
@@ -89,6 +89,52 @@ class ZStreamChunk[-R, +E, +A](val chunks: ZStream[R, E, Chunk[A]]) extends Seri
   final def catchAllCause[R1 <: R, E2, A1 >: A](f: Cause[E] => ZStreamChunk[R1, E2, A1]): ZStreamChunk[R1, E2, A1] =
     ZStreamChunk(chunks.catchAllCause(c => f(c).chunks))
 
+  /**
+   * Chunks the stream with specifed chunkSize
+   * @param chunkSize size of the chunk
+   */
+  final def chunkN(chunkSize: Int): ZStreamChunk[R, E, A] =
```

Comment:
I referenced this `chunkN` code from the `ZStream` itself and there also `checkSize` value check is not being taken care of, I believe this check is supposed to be taken care at the upper layer instead of at `chunkN`. I agree that we should check in the `chunkN` itself but wanted to keep the code as harmonized as possible. What do you think?